### PR TITLE
Add remove command

### PR DESCRIPTION
**Pull request by wengfaing**

### CODE REVIEW

**Comment by Wincenttjoi** [comment]
@wengfaing do you want to add the relevant unit/ integration tests first for this command? Including parser for the command, logic, etc. Or do you prefer to do these tests in next iteration?

**Comment by wengfaing** [comment]
> @wengfaing do you want to add the relevant unit/ integration tests first for this command? Including parser for the command, logic, etc. Or do you prefer to do these tests in next iteration?

I think i'll work on it in the next iteration!

**Comment by Wincenttjoi** [comment]
Sure, LGTM for this implementation! We can make the remove unit tests' tasks during next iteration

File: src/main/java/seedu/address/model/item/Quantity.java [hunk 1]
```diff
@@ -42,6 +42,16 @@ public Quantity add(Quantity quantity) {
         return new Quantity(Integer.toString(value));
     }
 
+    /**
+     * subtracts a Quantity's value from another quantity's value
+     * @param quantity another quantity
+     * @return Difference in Quantity
```
Comment (Wincenttjoi) [inline src/main/java/seedu/address/model/item/Quantity.java]: If I rmb correctly, the following word after @return it should be the type, then description for subsequent words

Comment (wengfaing) [reply src/main/java/seedu/address/model/item/Quantity.java]: Ok fixed in latest commit

**Pull request merged**